### ISSUE TITLE
Source Control goes to empty

### ISSUE THREAD

**Issue by rolivares** [microsoft/vscode#102652]
Issue Type: <b>Bug</b>

with no reason vscode stop on showing files with differences , even when is announcing that there are '(N)' changes on source control badge.

Sidebar is entirely empty and no files are visibles.

Some errors relates to 'Cannot read property XXX of null' are displayed when I try to change properties like 'show as tree/list' on top of the sidebar.

VS Code version: Code 1.47.1 (485c41f9460bdb830c4da12c102daff275415b53, 2020-07-13T22:42:53.386Z)
OS version: Windows_NT x64 10.0.18362

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz (4 x 2904)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.89GB (8.65GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (17)</summary>

Extension|Author (truncated)|Version
---|---|---
atlascode|atl|2.6.6
vscode-markdownlint|Dav|0.36.1
vscode-eslint|dba|2.1.7
gitlens|eam|10.2.2
vscode-html-css|ecm|0.2.3
prettier-vscode|esb|5.1.3
vscode-test-explorer|hbe|2.19.1
markdown-preview-bitbucket|hbr|1.0.0
vscode-gutter-preview|kis|0.26.2
remote-wsl|ms-|0.44.4
debugger-for-chrome|msj|4.12.9
vscode-extension-auto-import|Nuc|1.4.3
vscode-yaml|red|0.8.0
markdown-preview-enhanced|shd|0.5.12
open-in-browser|tec|2.0.0
vscode-wakatime|Wak|4.0.0
vscode-todo-highlight|way|1.0.4

</details>

**Comment by roblourens** [comment]
Can you share the full errors? You can see them all in the developer tools console (Help > Toggle Developer Tools)

**Comment by rolivares** [comment]
```
ERR Cannot read property 'domNode' of null: TypeError: Cannot read property 'domNode' of null
    at E.updateItemInDOM (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:512:636)
    at E._splice (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:507:744)
    at E.splice (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:506:699)
    at file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:30:994
    at Array.forEach (<anonymous>)
    at t.CombinedSpliceable.splice (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:30:981)
    at file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:547:565
    at t.EventBufferer.bufferEvents (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:98:214)
    at O.splice (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:547:532)
    at O.splice (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1097:878)
    at Object.splice (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:119:868)
    at t.IndexTreeModel.splice (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:104:116)
    at t.ObjectTreeModel._setChildren (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:111:499)
    at t.ObjectTreeModel.setChildren (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:111:371)
    at c._setChildren (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:117:159)
    at c.setChildren (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:116:409)
    at t.CompressibleObjectTreeModel.setChildren (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:120:771)
    at B.setChildren (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1114:255)
    at Oe.refresh (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:7241:820)
    at Oe._onDidSpliceRepositories (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:7239:1004)
    at Oe.setVisible (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:7240:984)
    at c.fire (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:95:821)
    at Ge.setVisible (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:6387:92)
    at file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:6405:328
    at Array.map (<anonymous>)
    at y.setVisible (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:6405:319)
    at i.setVisible (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:3903:24)
    at e.showComposite (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:5753:975)
    at e.doOpenComposite (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:5751:1007)
    at e.openComposite (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:5751:539)
    at e.doOpenViewlet (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:6346:900)
    at e.openViewlet (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:6346:313)
    at e.run (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:6178:913)
    at s.runAction (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:123:740)
    at s.run (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:123:598)
    at e.onClick (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:974:176)
    at Immediate._onImmediate (file:///D:/Users/rolivares/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.desktop.main.js:973:727)
    at processImmediate (internal/timers.js:439:21)
```

**Comment by rolivares** [comment]
I'm using workspace with three or four folders, each with their particular git repository

**Comment by gjsjohnmurray** [comment]
Maybe #102358 which is fixed in the upcoming 1.47.2.

**Comment by joaomoreno** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/102358 for which the fix is already out in Insiders, can you try?
